test/socket.jl

```diff
@@ -253,16 +253,27 @@ let
         bind(s, ip"0.0.0.0", 2000, reuseaddr = true, enable_broadcast = true)
         s
     end
+
+    function wait_until_timeout(recvs)
```

Comment:
Nitpicking - It is more of a `wait_with_timeout` rather than a `wait_until_timeout`.